src/biogeochem/VOCEmissionMod.F90

```diff
@@ -933,13 +890,28 @@ function get_gamma_T(t_veg240_in, t_veg24_in,t_veg_in, ct1_in, ct2_in, betaT_in,
     if ( (t_veg240_in > 0.0_r8) .and. (t_veg240_in < 1.e30_r8) ) then 
        ! topt and Eopt from eq 8 and 9:
        topt = co1 + (co2 * (t_veg240_in-tstd0))
-       Eopt = Ceo_in * exp (co4 * (t_veg24_in-tstd0)) * exp(co4 * (t_veg240_in -tstd0))
+       if ( (ivt_in == nbrdlf_dcd_brl_shrub) ) then  ! boreal-deciduous-shrub
+       ! coming from BEAR-oNS campaign willows results
+             Eopt = 7.9_r8 * exp (0.217_r8 * (t_veg24_in - tfrz - 24.0_r8))
+       else if ( (ivt_in == nc3_arctic_grass ) ) then  ! boreal-grass
+             Eopt = exp(0.12_r8 * (t_veg240_in - tfrz - 15._r8))
+       else
+             Eopt = Ceo_in * exp (co4 * (t_veg24_in - tstd0)) * exp(co4 * (t_veg240_in - tstd0))
+       endif
+
     else
        topt = topt_fix
        Eopt = Eopt_fix
     endif
     x = ( (1._r8/topt) - (1._r8/(t_veg_in)) ) / ct3
-    gamma_t_LDF = Eopt * ( ct2_in * exp(ct1_in * x)/(ct2_in - ct1_in * (1._r8 - exp(ct2_in * x))) )
+    ! for the boreal grass from BEAR-oNS campaign
+    if ( (ivt_in == nc3_arctic_grass ) ) then  ! boreal-grass
+        bet_arc_c3 = 95._r8 + 9.49_r8 * exp(0.53_r8 * (tfrz + 15._r8 - t_veg240_in))
```

Comment:
What about 95, 9.49, and 0.53?

Also the second one does not match the exact value in the intro to the PR. Is it a problem?

Reply:
95. is the standard activation energy for isoprene; 9.49 and 0.53 are the empirical parameters for the activation energy in response to 10-day temperature change.